nat_gateway/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "private_route_tables" {
```

Comment:
@MattiasGees I don't create any route table. That's just a route not a route table